src/server/kernel/wseat.cpp

```diff
@@ -404,7 +429,17 @@ void WSeatPrivate::on_start_drag(wlr_drag *drag)
         cursor->setDragSurface(wsurface);
     }
 }
-
+void WSeatPrivate::handleKeyEvent(QKeyEvent &e)
+{
+    Q_ASSERT(focusWindow);
+    if (e.type() == QEvent::KeyPress && QWindowSystemInterface::handleShortcutEvent(focusWindow,
```

Comment:
Why not use `qt_sendShortcutOverrideEvent`?

Reply:
That seems to be a internal interface for test use, `handleShortcut` is used by `QGuiApplication` and processes items' shortcut overridden, and here we only needs to send to window rather than child object. don't know which is correct.